src/js/themes/hpe.js

```diff
@@ -251,6 +251,20 @@ export const hpe = deepFreeze({
         },
         color: undefined,
       },
+      primary: {
+        // Designs call for using text-strong for the text color.
+        // Keeping the label color the same on hover to maintain readablility.
+        // color: hpe.global.colors['text-strong'].light,
```

Comment:
```suggestion
       color: 'text-strong',
```
In both light and dark mode, we want to ensure the label color stays as text-strong on hover